python/cudf/cudf/_lib/types.pyx

```diff
@@ -67,6 +67,7 @@ class TypeId(IntEnum):
     DECIMAL32 = <underlying_type_t_type_id> libcudf_types.type_id.DECIMAL32
     DECIMAL64 = <underlying_type_t_type_id> libcudf_types.type_id.DECIMAL64
     DECIMAL128 = <underlying_type_t_type_id> libcudf_types.type_id.DECIMAL128
+    STRUCT = <underlying_type_t_type_id> libcudf_types.type_id.STRUCT
```

Comment:
I really wish this entire enum were sorted to match `type_id` in libcudf... https://github.com/rapidsai/cudf/blob/cc1576590c4da2ba11112dd57bca25b5ea2e39bd/cpp/include/cudf/types.hpp#L196

Reply:
Done